inference-engine/samples/speech_sample/main.cpp

```diff
@@ -873,15 +936,23 @@ int main(int argc, char *argv[]) {
                             if (!useHetero) continue;
                             if (code != StatusCode::INFER_NOT_STARTED) continue;
                         }
-
+                        ConstOutputsDataMap newOutputInfo;
                         if (inferRequest.frameIndex >= 0) {
                             if (!FLAGS_o.empty()) {
                                 outputFrame =
                                         &ptrScores.front() + numScoresPerFrame * sizeof(float) * (inferRequest.frameIndex);
-                                MemoryBlob::CPtr moutput = as<MemoryBlob>(inferRequest.inferRequest.GetBlob(cOutputInfo.rbegin()->first));
+                                if (!outputs.empty()) {
```

Comment:
as i see only 1 outputs file will be written - an i correct? same with comparing to reference? shouldn't you then check that number of output names (-oname) == 1?